clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -128,6 +129,17 @@ def get_unschedulable_pods(
                 unschedulable_pods.append((pod, self._get_pod_unschedulable_reason(pod)))
         return unschedulable_pods
 
+    def freeze_agent(self, agent_id: str) -> None:
+        try:
+            body = {
+                "spec": {
+                    "taints": [{"effect": "NoSchedule", "key": "clusterman.yelp.com/terminating", "value": "True"}]
```

Comment:
ooh, sorry, final thing - it might be useful to set the value of this taint to a unix timestamp rather than just `True`? that way we can also see how long its been in that state?

Reply:
@Rob-Johnson  Yeah sure, it will be useful.

I tested it, kube-scheduler doesn't schedule new pods to tainted node as expected.
But kubectl command doesn't show "SchedulingDisabled" in status section. is it ok?
I think it is useful information. What do you think about change spec.unschedulable as well?

Reply:
> @Rob-Johnson Yeah sure, it will be useful.
> 
> I tested it, kube-scheduler doesn't schedule new pods to tainted node as expected. But kubectl command doesn't show "SchedulingDisabled" in status section. is it ok? I think it is useful information. What do you think about change spec.unschedulable as well?

I think that's fine - the taint is 'enough' to stop things being scheduled

Reply:
> it might be nicer for humans to have it be the taint value so that it's more visible 
agreed - if we can do both, then great

Reply:
timestamp was added to the value.